src/listeners.c

```diff
@@ -112,6 +112,9 @@ void start_ex_listener(state_t *st) {
     case KEY_BACKSPACE:
       handle_backspace(st);
       break;
+    case KEY_ESC:
+      st->mode = NORMAL;
+      break;
```

Comment:
https://github.com/Azhng/viw/blob/9ab4fff47ddce65744e524de9e860bb2db1fef96/src/state.c#L79 ma bad we are actually good here :) clear_row is run when `update_state` is run on each tick